doc/api/child_process.markdown

```diff
@@ -67,22 +67,22 @@ implement the Node.js [`EventEmitter`][] API, allowing the parent process to
 register listener functions that are called when certain events occur during
 the life cycle of the child process.
 
-The `child_process.exec()` and `child_process.execFile()` methods additionally
+The [`child_process.exec()`][] and [`child_process.execFile()`][] methods additionally
 allow for an optional `callback` function to be specified that is invoked
 when the child process terminates.
 
 ### Spawning `.bat` and `.cmd` files on Windows
 
-The importance of the distinction between `child_process.exec()` and
-`child_process.execFile()` can vary based on platform. On Unix-type operating
-systems (Unix, Linux, OSX) `child_process.execFile()` can be more efficient
+The importance of the distinction between [`child_process.exec()`][] and
+[`child_process.execFile()`][] can vary based on platform. On Unix-type operating
+systems (Unix, Linux, OSX) [`child_process.execFile()`][] can be more efficient
 because it does not spawn a shell. On Windows, however, `.bat` and `.cmd`
 files are not executable on their own without a terminal, and therefore cannot
-be launched using `child_process.execFile()`. When running on Windows, `.bat`
-and `.cmd` files can be invoked using `child_process.spawn()` with the `shell`
-option set, with `child_process.exec()`, or by spawning `cmd.exe` and passing
+be launched using [`child_process.execFile()`][]. When running on Windows, `.bat`
```

Comment:
Unrelated to this PR, but perhaps you can fix: turns out that .bat/.exe/.com **are** directly executable by spawn, but .cmd, .pl, .js etc. are not. This means that npm installed node scripts are not executable.